docs/pages/aim-projects.md

```diff
@@ -184,6 +184,34 @@ Our ‘People, Policy and Impact’ group will share our learning and influence
 
 * You can read more about the MELD-B project on their page at [University of Southampton - MELD-B](https://www.southampton.ac.uk/publicpolicy/support-for-policymakers/policy-projects/Current%20projects/meld-b.page) or in their first blog post which you can find here: [University of Southampton - MELD-B Introductory Blog](https://www.southampton.ac.uk/publicpolicy/support-for-policymakers/blogs/evidence-to-policy-blog/meld-b-blog.page)
 
+## CoMPuTE
+
+> The Complex Multiple long-term conditions Phenotypes, Trends, and Endpoints (CoMPuTE) project is funded by the National Institute for Health and Care Research (NIHR) under its Programme Grants for Applied Research Programme (NIHR204406). It is based on the University of Oxford, University of Leeds and University College London. 
```

Comment:
Would it be worth to add a half-sentence summary of what their research aims to do here too? I see that we have that for most if not all of the other consortia instead of spelling out what their acronym means 🙂